packages/core/core/src/utils/template_generator.js

```diff
@@ -162,7 +167,7 @@ class TemplateGenerator {
   }
 
   installTemplate(templatePath, name, installPackages, cb) {
-    utils.cd(templatePath);
+    shelljs.cd(templatePath);
```

Comment:
This is the only place `utils.cd()` was used so I removed it and had `installTemplate()` make use of `shelljs.cd()` directly.